src/NodeJS/Javascript/package.json

```diff
@@ -1,13 +1,12 @@
 {
     "private": true,
+    "type": "module",
```

Comment:
Is node correctly running the bundled files? Just wondering if it correctly detects the `package.json`, so it can see the `type: "module"`. Otherwise it would see the compiled `.js` files as CommonJS.

Reply:
This is only relevant during the build of the library itself. The compiled files are embedded in the library and are passed to nodejs as a string on the command line. For this reason we have to specify `--input-type=module` as well when calling node. We cannot depend on any package.json being present at the time of execution, because there could be none and also the compiled file is always ESM regardless of what the package.json in the CWD specifies.